File: pm_core_1/src/core/domain/entity/issue_type_scheme_entity.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package entity

type IssueTypeSchemeEntity struct {
	BaseEntity
	TenantID           int64   `json:"tenantId"`
	Name               string  `json:"name"`
	Description        *string `json:"description,omitempty"`
	DefaultIssueTypeID int64   `json:"defaultIssueTypeId"`

	Items []IssueTypeSchemeItemEntity `json:"items,omitempty"`
}

File: pm_core/src/main/java/serp/project/pmcore/domain/entity/IssueTypeSchemeEntity.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.pmcore.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class IssueTypeSchemeEntity extends BaseEntity {
    private Long tenantId;
    private String name;
    private String description;
    private Long defaultIssueTypeId;

    private List<IssueTypeSchemeItemEntity> items;
}
